lib/c-api/tests/wasmer_wasm.h

```diff
@@ -0,0 +1,13 @@
+// This header file is used only for test purposes! It is used by unit
+// test inside the `src/` directory for the moment.
+
+#include <assert.h>
+#include "../wasmer_wasm.h"
+
+// Wasmer-specific shortcut to quickly create a `wasm_byte_vec_t` from
+// a string.
+static inline void wasm_byte_vec_new_from_string(
```

Comment:
Perhaps our own functions shouldn't use the standard `wasm_` prefix? Maybe `wasmer_`?

Reply:
Good idea.